plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/webhooks/WebHooksDialog.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.webhooks;
+
+import org.eclipse.jface.dialogs.Dialog;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Shell;
+
+import com.openshift.restclient.model.IBuildConfig;
+
+/**
+ * @author Fred Bricon
+ * @author Andre Dietisheim
+ */
+public class WebHooksDialog extends Dialog {
+
+	private IBuildConfig buildConfig;
+
+	public WebHooksDialog(Shell parent, IBuildConfig buildConfig) {
+		super(parent);
+		this.buildConfig = buildConfig;
+	}
+
+	protected void configureShell(Shell shell) {
+		super.configureShell(shell);
+		shell.setText("Web Hooks");
```

Comment:
Probably to be consistent with https://docs.openshift.org/latest/dev_guide/builds.html#build-triggers we should use 'Webhook' or 'Webhook triggers'.  This would need to change in all text that is visible to the user.